packages/fuselage-ui-kit/src/blocks/VideoConferenceBlock/VideoConferenceBlock.tsx

```diff
@@ -105,6 +126,21 @@ const VideoConferenceBlock = ({
   const { data } = result;
   const isUserCaller = data.createdBy._id === userId;
 
+  const messageFooterText = getMessageFooterText(data.users.length);
+
+  const joinedUsernames = [...data.users]
+    .splice(0, MAX_USERS)
+    .map(({ name, username }) => (showRealName ? name : username))
```

Comment:
Name can be undefined here.

Reply:
yep, let's replace it with the hook we're isolating in `ui-client`